gen_gold_lf.py

```diff
@@ -144,7 +148,15 @@ def process_cat(fpath, vmax_opath, field=None):
 
             result.pprint()
 
-            scale = rand.meta['DDP1_d{}_VOLFRAC'.format(idx)]
+            rand_G9 = Table.read(os.environ['RANDOMS_DIR'] + '/randoms_bd_ddp_n8_G9_0.fits')
```

Comment:
Don't read in the randoms xxx times (once for each d8).

And some syntax sugar:
rands = [Table.read(os.environ['RANDOMS_DIR'] + '/randoms_bd_ddp_n8_G{}_0.fits') for field in [9, 12, 15]]

scale = np.array([x.meta['KEYWORD'] for x in rands]).mean()